app/components/action-feed-entry/styles.scss

```diff
@@ -8,6 +8,9 @@
 
 }
 
+// Used in Unit tests
+.action-body { } /* stylelint-disable-line block-no-empty */
```

Comment:
Why does the class need to be in the stylesheet? Do unused styles get removed on compile or something?

Reply:
css modules munges all the class names when the app is built. It's in the style sheet so the tests can get the "compiled" name. I could just leave it untouched, as a `class` rather than `local-class` but it just looks a bit odd. Or do a weird selector. Whichever works for me.

Reply:
I don't really have a preference. I'll leave that up to @alexschiller